coverage/Chrome 57.0.2987 (Mac OS X 10.12.3)/lcov-report/base.css

```diff
@@ -0,0 +1,213 @@
+body, html {
```

Comment:
The coverage folder is auto-generated after tests are run. This shouldn't be included in the repo. Please add `coverage` to the `.gitignore` file. 

You can revert the commit that included this.